README.md

```diff
@@ -43,9 +43,9 @@ This **open-source plugin was developed to help the Sylius community**. If you h
 
 ## Installation
 ```bash
-$ composer require bitbag/blacklist-plugin
+composer require bitbag/blacklist-plugin -W --no-scripts
```

Comment:
But... why `-W` ?

Reply:
Yesterday I got problems without that flag. But today it is ok. Done